source/software/matlab_parallel_computing.rst

```diff
@@ -3,12 +3,131 @@
+`repository <https://github.com/hpcleuven/matlab-remote>`_. Place this folder somewhere on your cluster storage (e.g. ``$VSC_DATA``). Set your working directory to
+the directory where you saved the scripts. Within the folder, you will see a ``mdcs.rc`` file. This is a configuration file that allows you to adapt the cluster
+profile to your needs. There is only one setting that needs to be changed with the approach you are using here: ``LocalJobStorageLocation``. This will define where
+your job output will be directed to. There is no need for quotation marks when defining this path. **Be careful**: when leaving this field blank, this location will 
+default to your ``$VSC_HOME``, which could result in quickly filling up your home directory.
+
+Next, you'll need to load the module, and start it. It is recommended to do this via an interactive job.
+
+::
+
+   $ # Genius
+   $ module use /apps/leuven/skylake/2021a/modules/all
+   $ module load MATLAB/R2022b #MATLAB/R2022a if you would like to load this version
+    
+Once the module is loaded, start Matlab. You can start it in a terminal without the GUI by using the ``-nodisplay`` flag. Then, run the ``configCluster`` script.
+This will set up the parallel configuration.  
+
+::
+
+   $ matlab -nodisplay
+
+::
+
+   >> rehash toolbox;
+   >> configCluster;
+   
+You will get a message as follows if it succeeded: 'Complete.  Default cluster profile set to "<cluster_name> <matlab_version>".'. 
+   
+Before you will be able to submit multi-node jobs to the cluster, you will first need to get a handle on the cluster, and then set the additional properties of the 
+loaded profile.
+
+::
+
+   >> % get a handle on the cluster
+   >> c = parcluster;
+   >> % If you would prefer to use the local resources (i.e. the resources of the node you are currently on), use:
+   >> c = parcluster('local');
+
+If you would have multiple cluster profiles, you can also load the correct cluster profile with:
+
+:: 
+
+   >> c = parcluster("<profile_name>") % for example "Genius R2022a"
+   
+You can set a range of additional properties on each cluster profile. There are two properties that are required, namely ``AccountName`` and ``WallTime``. Set any 
+other to your preference. Once these are saved, they will be kept in the settings of your cluster profile, but they can always be adapted again later.
+
+::
+
+   >> % View the additional properties
+   >> c.AdditionalProperties
+   >> % set the required properties
+   >> c.AdditionalProperties.AccountName = <account-name>;
+   >> c.AdditionalProperties.WallTime = '05:00:00';
+   >> % save the profile
+   >> c.saveProfile
+   
+Now the profile is ready, and you should be able to submit multi-node jobs with Matlab now.
+
+Submitting jobs
++++++++++++++++
+
+You can start both interactive and independent batch jobs with this profile. Here, we will show how to start these jobs from an opened Matlab session.
```

Comment:
Do I understand it correctly that Matlab submits these jobs for you?
And so that you always need to first launch Matlab in some way, on a login node,
and that those commands below will create/submit/process jobs?
If so, it would be good to mention this more explicitly, because this would be
quite different from standard batch jobs (where you just write a batch script
and then submit that batch script with qsub).

Reply:
You indeed need to do this from within Matlab. I will modify the explanation